packages/components/src/components/Avatar/index.tsx

```diff
@@ -55,10 +56,10 @@ export const Pro = styled(Text).attrs({ size: 1, weight: 'bold' })(
   })
 );
 
-export const Avatar = ({ user, ...props }: IAvatarProps) =>
+export const Avatar = ({ user, file, ...props }: IAvatarProps) =>
   user && (
     <AvatarContainer {...props}>
-      <AvatarImage src={user.avatarUrl} alt={user.username} />
+      <AvatarImage src={file || user.avatarUrl} alt={user.username} />
```

Comment:
**nitpick:** Assuming we want to fall back when `file` is `null` here, this may better align with intent. 😊 

```suggestion
      <AvatarImage src={file ?? user.avatarUrl} alt={user.username} />
```